src/main/resources/assets/gregtech/lang/en_us.lang

```diff
@@ -2958,4 +2959,4 @@ gregtech.command.util.hand.fluid=Fluid Info: %s %d / %d mb; Can Fill: %s; Can Dr
 gregtech.command.util.hand.material_meta_item=Ore Prefix: %s; Material: %s
 gregtech.command.util.hand.tool_stats=Tool Stats Class: %s
 gregtech.command.util.hand.meta_item=Meta Item Name: %s
-gregtech.command.util.hand.not_a_player=This command is only usable by a player.
+gregtech.command.util.hand.not_a_player=This command is only usable by a player.
```

Comment:
Please be mindful regarding file endings so there are no necessary changes.

Reply:
100% agree, this is a deadly sin :) I was nervous with that first PR I guess... I fixed the problem.